rubix-spi/src/main/java/com/qubole/rubix/spi/ClusterManager.java

```diff
@@ -153,28 +167,28 @@ public ClusterInfo getClusterInfo()
   {
```

Comment:
`getClusterInfo` can be simplified now. We don't need to expose nodes to anything outside ClusterManager now. Change this method to `getCurrentNodeName` and drop `ClusterInfo`

Reply:
Move the check of BKS `if (currentNodeIndex.isEmpty() || nodes == null) {` in this method now.